lib/Mad/Mapper.pm

```diff
@@ -154,18 +154,30 @@ It is also possible to define aliases for "%t", "%c", "%c=" and "%pc". Example:
 
 =cut
 
-sub expand_sst {
-  my ($self, $sst, @args) = @_;
+sub expand_sql {
+  my ($self, $sql, @args) = @_;
   my $p;
 
-  $sst =~ s|(?<!\\)\%c(?:\.(\w+))?\=|{$p = $1 ? "$1." : ""; join ',', map {"$p$_=?"} $self->columns}|ge;
-  $sst =~ s|(?<!\\)\%c\?|{join ',', map {"?"} $self->columns}|ge;
-  $sst =~ s|(?<!\\)\%c(?:\.(\w+))?|{$p = $1 ? "$1." : ""; join ',', map {"$p$_"} $self->columns}|ge;
-  $sst =~ s|(?<!\\)\%pc(?:\.(\w+))?|{$p = $1 ? "$1." : ""; join ',', map {"$p$_"} $self->pk, $self->columns}|ge;
-  $sst =~ s|(?<!\\)\%t(?:\.(\w+))?|{$self->table. ($1 ? " $1" : "")}|ge;
-  $sst =~ s|\\%|%|g;
+  $sql =~ s|(?<!\\)\%c(?:\.(\w+))?\=|{$p = $1 ? "$1." : ""; join ',', map {"$p$_=?"} $self->columns}|ge;
+  $sql =~ s|(?<!\\)\%c\?|{join ',', map {"?"} $self->columns}|ge;
+  $sql =~ s|(?<!\\)\%c(?:\.(\w+))?|{$p = $1 ? "$1." : ""; join ',', map {"$p$_"} $self->columns}|ge;
+  $sql =~ s|(?<!\\)\%pc(?:\.(\w+))?|{$p = $1 ? "$1." : ""; join ',', map {"$p$_"} $self->pk, $self->columns}|ge;
+  $sql =~ s|(?<!\\)\%t(?:\.(\w+))?|{$self->table. ($1 ? " $1" : "")}|ge;
+  $sql =~ s|\\%|%|g;
+
+  return $sql, @args;
+}
+
+=head2 expand_sst
+
+  ($sql, @args) = $self->expand_sst($sql, @args);
 
-  return $sst, @args;
+expand_sst() is deprecated in favor of L</expand_sql>().
```

Comment:
Please drop the example code and write "DEPRECATED". Also drop "()" after expand_sql.